easybuild/easyconfigs/g/GffCompare/GffCompare-0.11.6-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'GffCompare'
+version = '0.11.6'
+
+# Some docs also at GitHub site - this is referred to as canonical
+homepage = 'https://ccb.jhu.edu/software/stringtie/gffcompare.shtml'
+description = """GffCompare provides classification and reference annotation mapping and
+ matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+# Switched to GitHub sources owing to SSL error at original download site
+source_urls = ['https://github.com/gpertea/gffcompare/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['21d37a43b6db1deb30fc5bc5198121a0b65e33edd5be2f55ae6100a34ee5df16']
```

Comment:
The checksum in the existing 0.11.6 easyconfig is `6095d97301a9c8dbd5e99e8c42860ce5953546277a09a0b46c835019815f6174` and some sites will already have this download.

Reply:
Ah, you mean include both of them?

Reply:
Yes - I think we should have it like how it is in https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/m/MetaPhlAn2/MetaPhlAn2-2.7.8-foss-2020a-Python-3.8.2.eb#L21